inspector/src/inspector.ts

```diff
@@ -300,6 +316,11 @@ export class Inspector {
 
         this._CopyStyles(window.document, parentDocument);
 
+        setTimeout(() => {

```

Comment:
I would recommend to have the timeout on demand to avoid popping effect

Reply:
sorry, I don't know understand the concept. For this, you mean to set it like this?
```
const timeOut = setTimeout(() => {
this._CopyStyles(window.document, parentDocument);
}, 0);

```

Reply:
I mean either call the copy style directly or with a timeout based on a parameter of the function so no changes for current users of that function

Reply:
ah, got it.